frida_mode/Makefile

```diff
@@ -0,0 +1,137 @@
+PWD:=$(shell pwd)/
+INC_DIR:=$(PWD)inc/
+SRC_DIR:=$(PWD)src/
+INCLUDES:=$(wildcard $(INC_DIR)*.h)
+SOURCES:=$(wildcard $(SRC_DIR)*.c)
+BUILD_DIR:=$(PWD)build/
+
+FRIDA_BUILD_DIR:=$(BUILD_DIR)frida/
+FRIDA_TRACE:=$(FRIDA_BUILD_DIR)afl-frida-trace
+
+GUM_DEVKIT_ARCH=x86_64
+GUM_DEVKIT_VERSION=14.2.13
+GUM_DEVKIT_FILENAME=frida-gum-devkit-$(GUM_DEVKIT_VERSION)-linux-$(GUM_DEVKIT_ARCH).tar.xz
```

Comment:
This shouldn't be hardcoded to Linux.

Reply:
maybe `$(shell uname -s)`?

Reply:
So the current URL for macOS is:
```
https://github.com/frida/frida/releases/download/14.2.13/frida-gum-devkit-14.2.13-macos-arm64.tar.xz
```
Not `darwin`. =(

Reply:
I can still use uname for the platform detection and just throw in an if statement to convey what uname says to the right token in the file name?

Reply:
yes, that is the easy way.

Reply:
you can see that in the main afl++ GNUmakefile

Reply:
Fixed